### PR TITLE
Update CONTRIBUTING.md and drop bin/bootstrap

### PR DESCRIPTION
**Pull request by olleolleolle**
## Summary

Drop the `bin/bootstrap` file.

Copy-editing and spell-checking the CONTRIBUTING.md document, **to remove the reference to the bin/bootstrap** file.

## Details

As discussed in #582.

## Motivation and Context

`bundle install` covers the needs.

## Types of changes

- [x] Update documentation

## Checklist:

- [x] I have updated the documentation accordingly.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -21,11 +21,11 @@ We would love to get help from you as "user" and "contributor".
 The rest of this document is a guide for those maintaining Aruba, and others
 who would like to submit patches.
 
-## Contributing to the "aruba" project
+## Contributing to the `aruba` project
```
Comment (mvz) [inline CONTRIBUTING.md]: I'd go with just writing 'Aruba' to refer to the project. As in, no quotes, and with a capital 'A'.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -37,63 +37,62 @@ please learn our basic rules:
...
 
-**Contributors**
+#### Contributors
 
 Please...
 
 * Fork the project. Make a branch for your change.
 * Make your feature addition or bug fix &mdash; if you're unsure if your
-  addition will be accepted, open an issue for discussion first
+  addition will be accepted, open an Issue for discussion first
```
Comment (mvz) [inline CONTRIBUTING.md]: Maby use 'GitHub Issue' as above?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -37,63 +37,62 @@ please learn our basic rules:
...
-* Please do not change the Rakefile, version, or history.
-  (If you want to have your own version, that is fine but
-  bump version in a commit by itself so we can ignore it when we merge your change.)
-* Make sure your pull request complies with our development style.
-* Rebase your branch if needed to reduce clutter in our git history.
-* Make sure you don't break other people's code &mdash; On major changes: First
+* Do not change the `Rakefile`, gem version number in `version.rb`, or
+  [`CHANGELOG.md`][].
+  (If you want to have your own version, that is fine but bump version in a
+  separate commit, so that we can ignore that commit when we merge your change.)
```
Comment (mvz) [inline CONTRIBUTING.md]: :+1:

File: CONTRIBUTING.md (hunk 1 above)
Comment (olleolleolle) [reply CONTRIBUTING.md]: Thanks, done!

**Pull request merged**